x-pack/auditbeat/module/system/_meta/fields.yml

```diff
@@ -4,7 +4,25 @@
     These are the fields generated by the system module.
   release: experimental
   fields:
-    - name: system.audit
-      type: group
+
+  - name: event
+    type: group
+    fields:
+    - name: origin
+      type: keyword
       description: >
-      fields:
+        Origin of the event. This can be a file path (e.g. `/var/log/log.1`),
+        or the name of the system component that supplied the data (e.g. `netlink`).
+
+  - name: user
+    type: group
+    fields:
+    - name: terminal
```

Comment:
@ruflin Same here. I've also started using this in Filebeat's auditd module. I also think this is something we could consider in ECS.